packages/raystack/tsconfig.json

```diff
@@ -6,9 +6,10 @@
     "target": "esnext",
     "isolatedModules": false,
     "paths": {
-      "~/*": ["./*"]
+      "~/*": ["./*"],
+      "@raystack/apsara/v2": ["./v2/*"]
```

Comment:
why cant we use `~/v2` here instead of `@raystack/apsara/v2` ?

Reply:
`@raystack/apsara/v2` makes it more clear about where to import from.

Reply:
This is for internal code, why should someone import files from `@raystack/apsara/v2` inside the source of `@raystack/apsara` ?

Reply:
Changed to `~/v2/*`.

Reply:
@paanSinghCoder i think should use v1. since we have not launched stable version of apsara. and these changes will actually result in v1 since curent versions are still at 0.x cc @rsbh

Reply:
Makes sense! We started using the terms `v1` and `v2` from day one which brought us to this point. 

Moreover, In my opinion the new version might take some time to be fully stable. But it still makes more sense to call it v1 rather than v2.

Reply:
Renamed v2 to v1 everywhere. Merging.